test.js

```diff
@@ -67,10 +67,10 @@ test('should throw if the version number is not a string', t => {
   t.plan(1)
 
   try {
-    fp(() => {}, 12)
+    fp(() => {}, { version: 12 })
```

Comment:
Can we keep this test? Or this PR is a breaking change?

Reply:
At the moment the second parameter should be a string or object.